src/translations/common/titles.ts

```diff
@@ -2,7 +2,7 @@ import { translation as _ } from '@atb/translations/commons';
 import { orgSpecificTranslations } from '../utils';
 
 const TitlesInternal = {
-  siteTitle: _('AtB (nb)', 'AtB (en)', 'AtB  (nn)'),
+  siteTitle: _('AtB', 'AtB', 'AtB'),
```

Comment:
Might want to update all of these to "AtB Reisesøk", "FRAM Reisesøk", etc., so that we are in line with the sketches. That way it's easier for Julia to verify the design - I think

Reply:
What will that be in english? "FRAM Travel Search" etc.?

Reply:
I guess so! We can verify with Julia when we go through the implementation